libtaxii/__init__.py

```diff
@@ -52,8 +52,13 @@ def get_message_from_http_response(http_response, in_response_to):
     else:
         raise ValueError('Unsupported response type: %s.' % http_response.__class__.__name__)
 
+def return_check(return_message,message,object):
```

Comment:
This has a code smell to it - Can the function be refactored to always return the same type?